zipline/algorithm.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.algoscript = kwargs.pop('script', None)
 
         self._initialize = None
+        self._analyze = kwargs.pop('analyze', None)
```

Comment:
Yep, that's what I would have pointed to you, and sorry our support of windows is less than adequate...